README.md

```diff
@@ -34,3 +34,23 @@ A mod which enables Hollow Knight to act as an Archipelago client, enabling mult
 4. Enter in the correct settings for your Archipelago server.
 5. Hit **Start** to begin the game. The game will stall for a few seconds while it does all item placements.
 6. The game will immediately drop you into the randomized game. So if you are waiting for a countdown then wait for it to lapse before hitting Start, or hit Start then pause the game once you're in it.
+
+# Contributing
+Contributions are welcome, all code is licensed under MIT license. Please track your work within the repository if you are taking on a feature. This is done via GitHub Issues. If you are taking on an issue please comment on the issue and assign yourself (if possible). If you are looking to contribute something that isn't in the issues list then please submit an issue to describe what work you intend to take on.
+
+There are a few guidelines/rules I personally follow and deeply encourage all contributors to follow:
+* All issues must be linked to the [Hollow Knight Archipelago](https://github.com/users/Ijwu/projects/1/views/1) project board.
+* All issues must be linked to a release milestone.
+* All issues should be labeled appropriately.
+* All issues should be in the correct Kanban column to describe their current status.
+* If an issue is in `In Progress` status then it must have someone assigned to it.
+* Pull Requests must have a linked issue which they close out.
+
+Okay, if you read this far your eyes probably glazed over at the list above. It's... uh, just not very hard to track your work. Just refer to the list everytime you interact with the repository and go through the guidelines in order. It becomes second nature very quickly.
+
+**If you're not going to do any of the above then I at least ask you to create an issue describing any work you intend to take on, at the minimum.** I can handle the rest.
+
+## Development Setup
+You will need to alter the `.csproj` file to point to your local HK installation. Then you will need to make sure to not commit that change or I'll ding you in a PR about it. Line 15 will do it: https://github.com/Ijwu/Archipelago.HollowKnight/blob/main/Archipelago.HollowKnight/Archipelago.HollowKnight.csproj#L15
+
+Post-build events will automatically package the mod for export **as well as install it in your HK installation.** When developing on the mod **do not install Archipelago through Scarab.**
```

Comment:
If AP is installed through Scarab, it will end up in a different folder name and both versions of the mod will fail to load.

Maybe "If Archipelago is installed through Scarab, uninstall or disable it before testing.  Archipelago.HollowKnight will not load if both Scarab and development versions are installed at the same time.".  This probably needs cleanup too, but it's a start.

I always feel including a little of the "why" is useful for instructions like this.

Reply:
Coming in a following PR.